slackclient/slackrequest.py

```diff
@@ -44,33 +65,22 @@ def append_user_agent(self, name, version):
         else:
             self.custom_user_agent = [[name, version]]
 
-    def do(self, token, request="?", post_data=None, domain="slack.com", timeout=None):
+    def make_http_request(self, token=None, api_method="?", post_data=None, timeout=None):
```

Comment:
I renamed it back. We need to rename that method in the future and make it private.

Passing `token` overrides the client's own token, this is useful for things like the refresh request because it overrides or sets the auth header. This is currently allowed.